docs/Developers/Writing_Native_Bears.rst

```diff
@@ -52,13 +52,15 @@ each file:
 .. code:: python
 
     from coalib.bears.LocalBear import LocalBear
+    import logging
 
 
     class HelloWorldBear(LocalBear):
         def run(self,
                 filename,
                 file):
-            self.debug("Hello World! Checking file", filename, ".")
+            logging.debug("Hello World! Checking file {filename} .".format(
```

Comment:
Optional: You can omit the `filename` part inside `{...}`

Reply:
Makes the `format`-call shorter

Reply:
So it would be `"Hello World! Checking file {} .".format(filename)` instead if I understand correctly?

Reply:
yup @Kagamihime